samples/exemplar/src/main/kotlin/com/squareup/exemplar/actions/HelloWebAction.kt

```diff
@@ -8,22 +11,39 @@ import misk.web.ResponseContentType
 import misk.web.actions.WebAction
 import misk.web.mediatype.MediaTypes
 import okhttp3.Headers
+import java.util.Base64
 import javax.inject.Singleton
 
 @Singleton
-class HelloWebAction : WebAction {
+class HelloWebAction: WebAction {
+
+  @Inject @Named("helloNameKey") private lateinit var cipher: Cipher
+
   @Get("/hello/{name}")
   @ResponseContentType(MediaTypes.APPLICATION_JSON)
   fun hello(
     @PathParam name: String,
     @Suppress("UNUSED_PARAMETER")
     @RequestHeaders headers: Headers,
-    @QueryParam nickName: String?,
-    @QueryParam greetings: List<String>?
+    @QueryParam greetings: List<String>?,
+    @QueryParam action: String?
```

Comment:
Maybe the cipher example can be its own example. I think the HelloWebAction is supposed to be a bare minimum example.

Reply:
Yeah, encryption has no business here